frontend-react/src/components/header/GettingStartedDropdown.tsx

```diff
@@ -0,0 +1,51 @@
+import { Menu, NavDropDownButton } from "@trussworks/react-uswds";
+import { useEffect, useState } from "react";
+import { NavLink } from "react-router-dom";
+
+const GettingStartedDropdown = () => {
+    const [isOpen, setIsOpen] = useState(false);
+
+    /* Used since setIsOpen cannot be directly called in useEffect */
+    const handleClick = () => setIsOpen(false);
+    /* INFO
+       This has to be down on "mouseup" not "mousedown" otherwise clicking
+       any link in the list will result in the menu closing without registering
+       the click on the link; thus, you're not directed to the page desired */
+    useEffect(() => {
+        document.body.addEventListener("mouseup", handleClick);
+        return () => {
+            document.body.removeEventListener("mouseup", handleClick);
+        };
+    }, []);
```

Comment:
There's usually a "click outside" feature mixed in here. 
See: https://github.com/CDCgov/prime-simplereport/blob/main/frontend/src/app/utils/hooks.tsx#L7  
(Notice the parameters passed into the useEffect() at the end. `ref, onClickOutside`

Reply:
Looks like we have the same problem in the HowItWorksDropdown. Since it's existing I think it's ok to solve as a follow up